<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>ContactExaminerDlgUI</class>
 <widget class="QDialog" name="ContactExaminerDlgUI" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>289</width>
    <height>304</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Virtual Grasp</string>
  </property>
  <widget class="Q3GroupBox" name="groupBox5" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>160</y>
     <width>270</width>
     <height>90</height>
    </rect>
   </property>
   <property name="title" >
    <string>Virtual Grasp Quality</string>
   </property>
   <property name="orientation" >
    <enum>Qt::Vertical</enum>
   </property>
   <widget class="QLabel" name="qualityLabel" >
    <property name="geometry" >
     <rect>
      <x>71</x>
      <y>26</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text" >
     <string>0.0</string>
    </property>
    <property name="wordWrap" >
     <bool>false</bool>
    </property>
   </widget>
   <widget class="Line" name="line6" >
    <property name="geometry" >
     <rect>
      <x>120</x>
      <y>20</y>
      <width>20</width>
      <height>60</height>
     </rect>
    </property>
    <property name="frameShape" >
     <enum>QFrame::VLine</enum>
    </property>
    <property name="frameShadow" >
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="orientation" >
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="textLabel3" >
    <property name="geometry" >
     <rect>
      <x>27</x>
      <y>25</y>
      <width>40</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text" >
     <string>Quality:</string>
    </property>
    <property name="wordWrap" >
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="sampleButton" >
    <property name="geometry" >
     <rect>
      <x>150</x>
      <y>30</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text" >
     <string>Sample Quality...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="updateQualityButton" >
    <property name="geometry" >
     <rect>
      <x>30</x>
      <y>50</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text" >
     <string>Update</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="exitButton" >
   <property name="geometry" >
    <rect>
     <x>230</x>
     <y>260</y>
     <width>50</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text" >
    <string>Exit</string>
   </property>
  </widget>
  <widget class="Q3GroupBox" name="groupBox4" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>270</width>
     <height>140</height>
    </rect>
   </property>
   <property name="title" >
    <string>Contacts</string>
   </property>
   <property name="orientation" >
    <enum>Qt::Vertical</enum>
   </property>
   <widget class="QLabel" name="textLabel1" >
    <property name="geometry" >
     <rect>
      <x>120</x>
      <y>30</y>
      <width>49</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text" >
     <string>Marked:</string>
    </property>
    <property name="wordWrap" >
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="markedLabel" >
    <property name="geometry" >
     <rect>
      <x>160</x>
      <y>30</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text" >
     <string>0</string>
    </property>
    <property name="wordWrap" >
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="loadButton" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>100</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text" >
     <string>Load Contacts...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton" >
    <property name="geometry" >
     <rect>
      <x>110</x>
      <y>100</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text" >
     <string>Save Contacts...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearButton" >
    <property name="geometry" >
     <rect>
      <x>210</x>
      <y>100</y>
      <width>50</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text" >
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="markButton" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>20</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text" >
     <string>Mark Contact</string>
    </property>
   </widget>
   <widget class="QPushButton" name="showWrenchesButton" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>60</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text" >
     <string>Show wrenches</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3GroupBox</class>
   <extends>QGroupBox</extends>
   <header>Qt3Support/Q3GroupBox</header>
   <container>1</container>
   <pixmap></pixmap>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="global" >vector</include>
 </includes>
 <resources/>
 <connections>
  <connection>
   <sender>markButton</sender>
   <signal>clicked()</signal>
   <receiver>ContactExaminerDlgUI</receiver>
   <slot>markButton_clicked()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>loadButton</sender>
   <signal>clicked()</signal>
   <receiver>ContactExaminerDlgUI</receiver>
   <slot>loadButton_clicked()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveButton</sender>
   <signal>clicked()</signal>
   <receiver>ContactExaminerDlgUI</receiver>
   <slot>saveButton_clicked()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exitButton</sender>
   <signal>clicked()</signal>
   <receiver>ContactExaminerDlgUI</receiver>
   <slot>exitButton_clicked()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>updateQualityButton</sender>
   <signal>clicked()</signal>
   <receiver>ContactExaminerDlgUI</receiver>
   <slot>updateQualityButton_clicked()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sampleButton</sender>
   <signal>clicked()</signal>
   <receiver>ContactExaminerDlgUI</receiver>
   <slot>sampleButton_clicked()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearButton</sender>
   <signal>clicked()</signal>
   <receiver>ContactExaminerDlgUI</receiver>
   <slot>clearButton_clicked()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showWrenchesButton</sender>
   <signal>clicked()</signal>
   <receiver>ContactExaminerDlgUI</receiver>
   <slot>showWrenchesButton_clicked()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
